### PR TITLE
Fix get method invoked name

### PR DESCRIPTION
**Pull request by lukyanoffpashok**

### CODE REVIEW

File: aibolit/patterns/multiple_while/multiple_while.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
-import javalang
-
+from aibolit.utils.ast import AST, ASTNodeType
```
Comment (aravij) [inline aibolit/patterns/multiple_while/multiple_while.py:1]: This strange diff from another PR looks really weird. Can we update branches somehow to get rid of it?

File: aibolit/utils/ast.py [hunk 2]
```diff
@@ -186,9 +194,12 @@ def get_type(self, node: int) -> ASTNodeType:
     def get_method_invoked_name(self, invocation_node: int) -> MethodInvocationParams:
         assert(self.get_type(invocation_node) == ASTNodeType.METHOD_INVOCATION)
         # first two STRING nodes represent object and method names
-        object_name, method_name = islice(self.children_with_type(invocation_node, ASTNodeType.STRING), 2)
-        return MethodInvocationParams(self.get_attr(object_name, 'string'),
-                                      self.get_attr(method_name, 'string'))
+        children = list(self.children_with_type(invocation_node, ASTNodeType.STRING))
+        if len(children) == 1:
+            return MethodInvocationParams('inline method', self.get_attr(children[0], 'string'))
```
Comment (aravij) [inline aibolit/utils/ast.py:199]: I don't think there is a `inline method` term in Java. We should better make first line empty if we call for a local method.

File: aibolit/utils/ast.py [hunk 3]
```diff
@@ -186,9 +194,12 @@ def get_type(self, node: int) -> ASTNodeType:
     def get_method_invoked_name(self, invocation_node: int) -> MethodInvocationParams:
```
Comment (aravij) [inline aibolit/utils/ast.py:194]: We can start using this method in instance_of pattern.

**Comment by 0crat** [comment]
Job `gh:yegor256/aibolit#449` is not assigned, can't get performer

**Comment by 0crat** [comment]
Pull request [#449](https://github.com/yegor256/aibolit/issues/449) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

**Pull request closed without merging**